prboom2/src/r_drawflush.inl

```diff
@@ -161,12 +196,8 @@ static void R_FLUSHQUAD_FUNCNAME(void)
    byte *dest = drawvars.topleft + commontop*drawvars.pitch + startx;
    int count;
 #if (R_DRAWCOLUMN_PIPELINE & RDC_FUZZ)
-   int fuzz1, fuzz2, fuzz3, fuzz4;
-
-   fuzz1 = fuzzpos;
-   fuzz2 = (fuzz1 + tempyl[1]) % FUZZTABLE;
-   fuzz3 = (fuzz2 + tempyl[2]) % FUZZTABLE;
-   fuzz4 = (fuzz3 + tempyl[3]) % FUZZTABLE;
+   // Only whole flushes are supported for fuzz
+   return;
```

Comment:
Why was code here before but now there's nothing? What changed?

Reply:
I removed code that wasn't used by the new algorithm. There are 3 flush types: a whole flush which draws it column by column, a quad flush which draws 4 pixels in the same row at a time, and lastly a headtail flush which draws the pixels above and below the region a quad flush covers. You'll notice in the headtail flush that I put the same comment, only there it's a wrapper for the whole flush. Headtail flush and quad flush are called in sequence so they can't both be wrappers.